scripts/lock_requirements.sh

```
@@ -1 +1 @@
-poetry export -f requirements.txt --output requirements.txt --without-hashes
\ No newline at end of file
+uv export --format requirements.txt --without-hashes -o requirements.txt
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)